infrastructure/labelmaker/labeltester.jinja.schema

```diff
@@ -0,0 +1,52 @@
+info:
+  title: Label Maker Tester
+  author: mwallman@google
+  description: |
+    Configuration to test labelmaker with different disk types
+  version: 0.1
+
+imports:
+  - path: labeltester.jinja
+
+required:
+  - zone
+  - network
+  - subnetwork
+  - machineType
+  - blankImage
+  - serviceAccount
+  - labelmaker
+
+properties:
+  zone:
+    type: string
+    description: i.e. us-east1-d
+  
+  network:
+    type: string
+    description: Network uri
+
+  subnetwork:
+    type: string
+    description: Subnet uri
+
+  sourceImage: 
+    type: string
+    description: Image uri
+    default: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/ubuntu-1404-trusty-v20170202a
```

Comment:
please can you use a link to the image family? 

sourceImage: https://www.googleapis.com/compute/v1/projects/ubuntu-os-cloud/global/images/family/ubuntu-1604-lts

This ensures that when we release a new image due to security patches those that run this script won't be starting an insecure image.